src/main/app/forms/models/itemInMultiRowForm.ts

```diff
@@ -0,0 +1,6 @@
+export abstract class ItemInMultiRowForm {
+
+  abstract deserialize (input?: any): ItemInMultiRowForm
```

Comment:
There's no reason to specify this method here. Any class extending MultiRowForm has to know how to create its items anyway.